pkg/dashboard/router.go

```diff
@@ -60,13 +69,13 @@ func GetRouter(setters ...Option) *mux.Router {
 	// root
 	router.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		// catch all other paths that weren't matched
-		if r.URL.Path != "/" {
+		if r.URL.Path != "/" && r.URL.Path != opts.basePath {
```

Comment:
Ah, this is it. Can we modify this to include `/basepath` without the trailing slash?